lib/spritesheet.js

```diff
@@ -83,11 +84,17 @@ module.exports = function Spritesheet(name, scaledSprites, groupConfig, config,
         that.outputFilesize = data.outputFilesize;
     }
 
+     function calculateHash() {
+        var hash = crypto.createHash("sha1");
+        hash.update(_.pluck(scaledSprites, "path").sort().join(" ") + "_");
+        hash.update(that.extension + "_");
+        hash.update((groupConfig.quality ? groupConfig.quality : "noquality") + "_");
+        hash.update((groupConfig.compression ? groupConfig.compression : "nocompression"));
```

Comment:
doesn't update override the data on the hash? wouldn't this mean that the only 'update' that is actually relevant? (the rest would be overriden by the last one)